docs/operations/security-overview.md

```diff
@@ -23,66 +23,109 @@ title: "Security overview"
   -->
 
 
-## Overview
+
+This document provides an overview of Apache Druid security features, configuration instructions, and some best practices to secure Druid.
 
 By default, security features in Druid are disabled, which simplifies the initial deployment experience. However, security features must be configured in a production deployment. These features include TLS, authentication, and authorization.
 
-To implement Druid security, you configure authenticators and authorizers. Authenticators control the way user identities are verified, while authorizers map the authenticated users (via user roles) to the datasources they are permitted to access. Consequently, implementing Druid security also involves considering your datasource scheme, since that scheme represents the granularity at which data access permissions are allocated. 
 
-The following graphic depicts the course of request through the authentication process: 
+## Best practices
```

Comment:
Thanks for putting these best practices together. Reading through them, I made a bunch of comments. Here is some structure that I think will help.

We should break this into a few sections
1. Druid cluster set up
1a. Run as a non root user
1b. Disable javascript
1c. Enable authentication  and authorization
2.  Network setup
2a. Enable TLS
2b. Limit access to the cluster, etc.
3. Authentication + Authorization
3a. Points about authentication and authorization, minimum privileges, etc.